tools/outline_proxy_controller/outline_error.h

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// This file contains the C++11 standard conforming implementation of
+// all errors used by outline.
+
+#pragma once
+
+#include <system_error>
+
+namespace outline {
+
+/**
+ * @brief The standard error code constants used by outline.
+ * @remarks The codes are copied from "/src/www/model/errors.ts".
+ */
+enum class outline_errc {
```

Comment:
Check the naming convention, I think it should start with capital letter: https://google.github.io/styleguide/cppguide.html#General_Naming_Rules